src/main/java/com/uid2/shared/store/parser/SiteParser.java

```diff
@@ -23,7 +23,9 @@ public ParsingResult<Map<Integer, Site>> deserialize(InputStream inputStream) th
             JsonObject siteSpec = sitesSpec.getJsonObject(i);
             final int siteId = siteSpec.getInteger("id");
             final String name = siteSpec.getString("name");
+            final String description = siteSpec.getString("description");
             final Boolean enabled = siteSpec.getBoolean("enabled", false);
+            final Boolean visible = siteSpec.getBoolean("visible", true);
```

Comment:
I think `true` is a reasonable default. Only internal sites and special case sites should not be `visible`

Reply:
Should we use ObjectMapper here instead? So that there's no need to manually update this parser each time, and we can have the default value of visible in the class

Reply:
Yep let's use `true`